README.md

```diff
@@ -1,28 +1,54 @@
-# Magic Shell Linter Action
+# Shell Linter
 
-This action executes lint check on bash scripts
+A GitHub Action that performs static analysis for shell scripts using [ShellCheck](https://github.com/koalaman/shellcheck).
 
-<br>
-
-## Inputs
-
-### `path`
-
-Required. Path to the bash script.
+![](docs/images/preview.png)
 
 <br>
 
-## Example usage
+## Usage
 
+Run static analysis for all of the bash/sh scripts.
 ```yml
 jobs:
-  # Run job that executes the bash script
   lint:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v1
       - name: Lint check
-        uses: azohra/magic-shell-linter-action@master
+        uses: azohra/shell-linter@master
```

Comment:
When we are ready, we'll release the first version v0.1.0, so let's use that instead of @master.